examples/js-avo-inspector-example/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
awesome 😄

Reply:
`react-scripts` package is using eslint v6.6.0, but Avo Inspector uses eslint v7.4.0 and due to this project cannot be build. Other listed solutions did not resolve this except this one. But can check more of course if needed

Reply:
I was serious, we were using this solution by appending manually every time.

Reply:
oh, ok 😃